packages/desktopjs/src/Default/default.ts

```diff
@@ -260,6 +260,10 @@ export namespace Default {
             return new DefaultMessageBus(this);
         }
 
+        public setOptions(options: any) {
+            throw new TypeError("SetOptions not implemented by this container");
```

Comment:
I think this can just be empty no-op?  That way if your app is multicontainer targeted you can have options that span electron/openfin etc and have it still invoke browser and noop

Reply:
I agree the default would be no-op. However, on our side we have our own container, so we will have to do that override, so it will be good to fix this here :)

Reply:
@sophophilic PR is pending this update.  Everything else is good to go.